docs/advanced/customization-guide.md

```diff
@@ -491,6 +491,7 @@ The following features are available for matching:
 | **`cpu.security`** | attribute  |          |            | Features related to security and trusted execution environments
 |                  |              | **`sgx.enabled`** | bool | `true` if Intel SGX (Software Guard Extensions) has been enabled, otherwise does not exist
 |                  |              | **`se.enabled`** | bool  | `true` if IBM Secure Execution for Linux is available and has been enabled, otherwise does not exist
+|                  |              | **`tdx.enabled`** | bool | `true` if Intel TDX (Trusted Domain Extensions) is available and has been enabled, otherwise does not exist
```

Comment:
while this follows how it's written for `se.enabled`, I think it'd be good to clarify this label is available on the host and not that it's a "node is a TD":
```suggestion
|                  |              | **`tdx.enabled`** | bool | `true` if Intel TDX (Trusted Domain Extensions) is available on the host and has been enabled, otherwise does not exist
```

ditto for `features.md`.